modules/virt-about-using-virtual-gpus.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-using-virtual-gpus_{context}"]
+= About using virtual GPUs with {VirtProductName}
+
+If you have a host with a compatible graphics processing unit (GPU), you can use it to create virtual GPUs (vGPUs). {VirtProductName} can automatically create vGPUs and other _mediated devices_ if an administrator provides configuration details in the `HyperConverged` custom resource (CR). This automation is especially useful for large clusters.
```

Comment:
> If you have a host with a compatible graphics processing unit (GPU), you can use it to create virtual GPUs (vGPUs).

I was struggling a bit with this sentence as only specific vGPU enabled cards can be used. But I think the `Note` addresses this matter. Thanks!

Reply:
I changed this first sentence to "Some graphics processing unit (GPU) cards support the creation of virtual GPUs (vGPUs)." Does that sound better? Thanks so much @vladikr !